### ISSUE TITLE
[Spotlight] London Collection, Part 2 browse

### ISSUE THREAD

**Issue by devangm** [TAMULib/exhibits#103]
**URL:** https://spotlight.library.tamu.edu/spotlight/london-maps-batch-2/browse/london-collection-part-2
**WCAG 2.1 AA compliant? :** No
**Violation:** 1.1.1 Non-Text Content (A)
**Issues:** Decorative images with alt text
**Notes:** Within results navigation each element has multiple clicks to get to the next element and each redundant click allows interaction with the element. ex. "link graphic slide show, link slide show".

**Comment by qtamu** [comment]
Please double check with screen to make sure the labels for these are incorrect.

**Comment by qtamu** [comment]
The <desc> element is able to be removed with update of the blacklight-gallery gem. However, because that gem is dependent on other outdated gems the proper updated gem breaks the slideshow view.
